src/internal/kopia/conn_test.go

```diff
@@ -27,9 +28,10 @@ func openLocalKopiaRepo(
 	ctx context.Context, //revive:disable-line:context-as-argument
 ) (*conn, error) {
 	st := storeTD.NewFilesystemStorage(t)
+	hashStr := uuid.NewString()[:7]
 
 	k := NewConn(st)
-	if err := k.Initialize(ctx, repository.Options{}, repository.Retention{}); err != nil {
+	if err := k.Initialize(ctx, repository.Options{}, repository.Retention{}, hashStr); err != nil {
```

Comment:
I'm starting to feel like we could probably have `hashStr` inside the `conn` struct. Any thoughts @ryanfkeepers ?